papers/bibtex.py

```diff
@@ -35,6 +35,22 @@
 # Others is a reserved Bibtex keyword
 OTHERS_RE = re.compile(r'( and )?\s*others\.?\s*$', re.IGNORECASE | re.UNICODE)
 
+PAPER_TYPE_TO_BIBTEX = [
+   ('journal-article', 'article'),
+   ('proceedings-article', 'inproceedings'),
+   ('book-chapter', 'book'),
+   ('book', 'book'),
+   ('journal-issue', 'book'),
+   ('proceedings', 'proceedings'),
+   ('reference-entry', 'misc'),
+   ('poster', 'misc'),
+   ('report', 'article'),
+   ('thesis', 'phdthesis'),
+   ('dataset', 'misc'),
+   ('preprint', 'article'),
+   ('other', 'misc'),
+]
+
```

Comment:
Is there any particular reason to use a list of pairs rather a dict for `PAPER_TYPE_TO_BIBTEX`? The order of the items does not seem relevant and having a dict would simplify the lookup in the translation, no?

Reply:
I just stupidly copy/pasted the `PAPER_CHOICES` constant (which could probably be a dict as well ^^). Will fix.

Reply:
Yeah, for PAPER_CHOICES I was not aware of ordered dicts, which would be the right thing…